_field-types/mapping-parameters/index-options.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: Index options
+parent: Mapping parameters
+grand_parent: Mapping and field types
+nav_order: 70
+has_children: false
+has_toc: false
+---
+
+# Index options
+
+The `index_options` mapping parameter controls the level of detail stored in the inverted index for text fields. This setting directly influences both the index size and the capabilities available for scoring, phrase matching, and highlighting.
+
+The `index_options` parameter has the following valid values.
+
+| Value     | Stores                          | Description |
+|------------|----------------------------------|-------------|
+| `docs`     | Document IDs only               | Indexes only the existence of a term in documents. Does not store frequency or position. Minimizes index size; suitable for simple existence checks. |
```

Comment:
First sentence in the description: Should "documents" be either "a document" or "a set of documents"?